deploy/install.sh

```diff
@@ -7,33 +7,74 @@ if [ $# -ge 1 ] ; then
 fi
 
 NS=abis
+MDSNS=mds
 CHART_VERSION=0.0.1-develop
 
-echo Create $NS namespace
+echo "Create $NS namespace"
 kubectl create ns $NS
 
+echo "Create $MDSNS namespace"
+kubectl create ns $MDSNS
+
 function mock() {
-  echo Istio label
+  echo "Istio label for $NS"
   kubectl label ns $NS istio-injection=enabled --overwrite
+
   helm repo update
 
   echo "Copy configmaps"
   sed -i 's/\r$//' copy_cm.sh
   ./copy_cm.sh
 
-  echo Istio label
-  kubectl label ns $NS istio-injection=enabled --overwrite
-  helm repo update
+  echo "Configuring mock-mds"
+
+  # Additional configuration for mock-mds
+  echo "Copy secrets"
+  sed -i 's/\r$//' copy_secrets.sh
+  ./copy_secrets.sh
+
+  read -p "Provide mockmds bucket name: " s3_bucket
+  if [[ -z $s3_bucket ]]; then
+    echo "s3_bucket not provided; EXITING;"
+    exit 1
+  fi
+  if [[ $s3_bucket == *[' !@#$%^&*()+']* ]]; then
+    echo "s3_bucket should not contain spaces / any special character; EXITING"
+    exit 1
+  fi
 
-  echo Installing mock-mv
-  helm -n $NS install mock-mv mosip/mock-mv --version $CHART_VERSION
+  read -p "Provide mockmds s3 bucket region: " s3_region
+  if [[ $s3_region == *[' !@#$%^&*()+']* ]]; then
+    echo "s3_region should not contain spaces / any special character; EXITING"
+    exit 1
+  fi
 
-  echo Installing mock-abis
-  helm -n $NS install mock-abis mosip/mock-abis --version $CHART_VERSION
+  read -p "Provide S3 URL: " s3_url
+  if [[ -z $s3_url ]]; then
+    echo "s3_url not provided; EXITING;"
+    exit 1
+  fi
 
-  kubectl -n $NS  get deploy -o name |  xargs -n1 -t  kubectl -n $NS rollout status
+  s3_user_key=$(kubectl -n s3 get cm s3 -o json | jq -r '.data."s3-user-key"')
```

Comment:
what about `s3-user-secret` ?

Reply:
we are sharing s3 secrets via helm and copy_secret.sh @syedsalman3753